### DISCUSSION TITLE
MAUI Application

### DISCUSSION THREAD

**Discussion by Orgbrat** [wieslawsoltes/Svg.Skia#83] [Q&A]
Does anyone know if Svg.Skia  is compatible with a .NET Maui application?

Orgbrat

**Comment by wieslawsoltes** [comment]
If it supports SkiaSharp you can render svg there. If you want native MAUI control there is no support right now in Svg.Skia repo.

**Comment by wieslawsoltes** [comment]
Implemented MAUI support in https://github.com/wieslawsoltes/Svg.Skia/pull/508
